lib/src/sync-process/event.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright 2021 Google Inc. Use of this source code is governed by an
```

Comment:
Nit: I think we're supposed to use "Google LLC" instead of "Google Inc" for new license headers